examples/pipelines/finetune_stable_diffusion/components/clip_retrieval_component/Dockerfile

```diff
@@ -1,14 +1,31 @@
-FROM europe-west1-docker.pkg.dev/storied-landing-366912/storied-landing-366912-default-repository/mlpipelines/kubeflow/components/base_component:latest
+FROM --platform=linux/amd64 python:3.8-slim
```

Comment:
Can we remove `--platform=linux/amd64` and only have it as a flag in the bash script?